### ISSUE TITLE
dmg doesn't open any window on OS X 10.9.4 (Mavericks)

### ISSUE THREAD

**Issue by mig5** [TrsstProject/trsst#24]
I am attempting to run the .dmg of your latest release 0.2-SNAPSHOT, on OS X 10.9.4 (Mavericks).

But it doesn't actually load any window.. just appears as running in the toolbar.

The system log says only:

25/08/2014 3:35:27.787 pm trsst[46224]: log4j:WARN No appenders could be found for logger (com.trsst.Common).
25/08/2014 3:35:27.787 pm trsst[46224]: log4j:WARN Please initialize the log4j system properly.

Same messages appear when I run it from the CLI with:

java -jar Java/trsst-client-0.2-SNAPSHOT-jfx.jar serve --gui

Any idea how to clear this error?

Thanks!

**Comment by mmpowers** [comment]
Sounds like a problem with the MacGap wrapper on Mavericks.  I'm on 10.8 but overdue for updating so I'll have to do that to further diagnose.  

When you run from the CLI, do you see a line in the output like "Services now available at: https://172.20.1.37:60140/feed" or something similar? 

If so, you can just connect your local web browser to that URL (which is all the MacGap wrapper actually does with a WebView).  

Please let me know if that works -- thanks.

**Comment by mig5** [comment]
No, when I run from CLI I get just the log4j messages, and nothing else:

Miguels-MacBook-Pro:Contents miguel$ java -jar Java/trsst-client-0.2-SNAPSHOT-jfx.jar serve --gui
log4j:WARN No appenders could be found for logger (com.trsst.Common).
log4j:WARN Please initialize the log4j system properly.

That's it - nothing else happens.

Let me know if I can provide any other debug info. Thanks!
